scimono-compliance-tests/src/main/java/com/sap/scimono/scim/system/tests/launcher/TestsLauncher.java

```diff
@@ -25,24 +26,35 @@ private TestsLauncher(final LauncherProperties launcherProperties) {
 
   public void launch() {
     setTestParameters();
+  
+    LauncherDiscoveryRequest launcherDiscoveryRequest = createLauncherDiscoveryRequest();
+  
+    LegacyXmlReportGeneratingListener xmlReporter = new LegacyXmlReportGeneratingListener(FileSystems.getDefault().getPath("."),
+        new PrintWriter(System.out));
+    Launcher launcher = LauncherFactory.create();
+    launcher.discover(launcherDiscoveryRequest);
 
+    TestsResultListener resultListener = new TestsResultListener();
+    launcher.registerTestExecutionListeners(new TestsExecutionReporter(), resultListener, xmlReporter);
+    launcher.execute(launcherDiscoveryRequest);
+    
+    if(resultListener.isAnyTestFailed()) {
```

Comment:
Let's make this optional for anybody relying on current behavior e.g. introduce a new `com.sap.scimono.scim.system.tests.launcher.LauncherProperties.ParamName` e.g. "tests.listeners.enabled=FailedTestResultListener"